config/jobs/test-infra-hosts/jenkins/LinuxHostBuild.Jenkinsfile

```diff
@@ -0,0 +1,234 @@
+
+                        sleep 15s
+
+                        az vm run-command invoke \
+                            --resource-group ${VM_RESOURCE_GROUP}  \
+                            --name ${VM_NAME} \
+                            --command-id RunShellScript \
+                            --scripts "cd /home/jenkins/ && \
+                            git clone https://github.com/openenclave/test-infra && \
+                            cd test-infra && git checkout master"
+
+                        sleep 15s
+
+                        az vm run-command invoke \
+                            --resource-group ${VM_RESOURCE_GROUP}  \
+                            --name ${VM_NAME} \
+                            --command-id RunShellScript \
+                            --scripts 'bash /home/jenkins/test-infra/scripts/ansible/install-ansible.sh'
+
+                        sleep 15s
+
+                        az vm run-command invoke \
+                            --resource-group ${VM_RESOURCE_GROUP}  \
+                            --name ${VM_NAME} \
+                            --command-id RunShellScript \
+                            --scripts 'ansible-playbook /home/jenkins/test-infra/scripts/ansible/oe-contributors-acc-setup.yml'
+
+                        sleep 15s
+                        '''
+                    )  
+                }
+            }
+        }
+
+        stage('Save VM State') {
+            steps{
+                script{
+                    sh(
+                        script: '''
+                        az vm deallocate \
+                            --resource-group ${VM_RESOURCE_GROUP} \
+                            --name ${VM_NAME}
+
+                        az vm generalize \
+                            --resource-group ${VM_RESOURCE_GROUP} \
+                            --name ${VM_NAME}
+                        
+                        img_id=\$(az image create \
+                            --resource-group ${VM_RESOURCE_GROUP} \
+                            --name myImage \
+                            --source ${VM_NAME} \
+                            --hyper-v-generation V2 | jq -r '.id')
+
+                        az sig image-definition create \
+                            --resource-group ACC-Images \
+                            --gallery-name ACC_Images \
+                            --gallery-image-definition ${GALLERY_DEFN} \
+                            --publisher ${PUBLISHER} \
+                            --offer ${OFFER} \
+                            --sku ${SKU} \
+                            --os-type Linux \
+                            --os-state generalized \
+                            --hyper-v-generation V2 || true
+
+                        YY=$(date +%Y)
+                        DD=$(date +%d)
+                        MM=$(date +%m)
+
+                        GALLERY_IMAGE_VERSION="$YY.$MM.$DD${BUILD_ID}"
+                        GALLERY_NAME="ACC_Images"
```

Comment:
This variable might be better in the `environment` block so it's easy to find

Reply:
Ah, yes. Thanks, I changed the versioning scheme and missed this refactor. Doing so now.